### PR TITLE
Addtion pillow1 , Deletion celery0

### PR DESCRIPTION
**Pull request by NitigyaS**
##### SUMMARY

* Adding Pillow1 with bionic image
* Removing celery0 with trusty image

##### ISSUE TYPE

- Change Pull Request

##### COMPONENT NAME

Terraform
##### ADDITIONAL INFORMATION

```paste below
Terraform will perform the following actions:

  - module.openvpn.aws_ami.openvpn_image

  - module.server__celery0-staging.aws_instance.server

  ~ module.server__hwarehouse0-staging.aws_instance.server
      credit_specification.#:                    "1" => "1"
      credit_specification.0.cpu_credits:        "standard" => "unlimited"

  + module.server__pillow1-staging.aws_instance.server
      id:                                        <computed>
      ami:                                       "ami-012fd5eb46f56731f"
      arn:                                       <computed>
      associate_public_ip_address:               <computed>
      availability_zone:                         <computed>
      cpu_core_count:                            <computed>
      cpu_threads_per_core:                      <computed>
      credit_specification.#:                    "1"
      credit_specification.0.cpu_credits:        "unlimited"
      disable_api_termination:                   "true"
      ebs_block_device.#:                        <computed>
      ebs_optimized:                             "true"
      ephemeral_block_device.#:                  <computed>
      get_password_data:                         "false"
      host_id:                                   <computed>
      instance_state:                            <computed>
      instance_type:                             "t3.xlarge"
      ipv6_address_count:                        <computed>
      ipv6_addresses.#:                          <computed>
      key_name:                                  "nsharma"
      network_interface.#:                       <computed>
      network_interface_id:                      <computed>
      password_data:                             <computed>
      placement_group:                           <computed>
      primary_network_interface_id:              <computed>
      private_dns:                               <computed>
      private_ip:                                <computed>
      public_dns:                                <computed>
      public_ip:                                 <computed>
      root_block_device.#:                       "1"
      root_block_device.0.delete_on_termination: "true"
      root_block_device.0.volume_id:             <computed>
      root_block_device.0.volume_size:           "80"
      root_block_device.0.volume_type:           "gp2"
      security_groups.#:                         <computed>
      source_dest_check:                         "false"
      subnet_id:                                 "subnet-06b16804ca8c1e379"
      tags.%:                                    "3"
      tags.Environment:                          "staging"
      tags.Group:                                "pillowtop:bionic"
      tags.Name:                                 "pillow1-staging"
      tenancy:                                   <computed>
      volume_tags.%:                             "4"
      volume_tags.Environment:                   "staging"
      volume_tags.Group:                         "pillowtop:bionic"
      volume_tags.Name:                          "vol-pillow1-staging"
      volume_tags.ServerName:                    "pillow1-staging"
      vpc_security_group_ids.#:                  "3"
      vpc_security_group_ids.222337278:          "sg-0b622c7832a915d81"
      vpc_security_group_ids.3649618601:         "sg-088f7187d50e07de3"
      vpc_security_group_ids.64875667:           "sg-0d29a3bab8ae8a2a2"


Plan: 1 to add, 1 to change, 2 to destroy.
```

### CODE REVIEW

**Comment by snopoke** [comment]
I'm curious about these changes which seem unrelated to the PR:
```
  - module.openvpn.aws_ami.openvpn_image

  ~ module.server__hwarehouse0-staging.aws_instance.server
      credit_specification.#:                    "1" => "1"
      credit_specification.0.cpu_credits:        "standard" => "unlimited"
```

**Comment by NitigyaS** [comment]
I am not sure about the OpenVPN one, warehouse is there since it might not have been rolled out after pushing to master

File: environments/staging/inventory.ini.j2 [hunk 1]
```diff
@@ -86,9 +83,13 @@ celery1
 [pillow0]
 {{ pillow0-staging }} hostname=pillow0-staging ec2=yes
 
+[pillow1]
+{{ pillow1-staging }} hostname=pillow1-staging ec2=yes
+
 [pillowtop:children]
 # Amazon EC2
 pillow0
+pillow1
```
Comment (dannyroberts) [inline environments/staging/inventory.ini.j2]: You have to run aws-fill-inventory [--cached] again to sync this change to inventory.ini

**Comment by dannyroberts** [comment]
@snopoke Those apparent diffs are an defect of my terraform code, but actually represent noop changes. It's definitely an anti-pattern to have consecutive runs of `apply` show a diff, and I've been delinquent in cleaning these two known noop diffs up. I'm familiar with both.

The openvpn I think I've recently been able to fix by upgrading the aws provider plugin, which I did in https://github.com/dimagi/commcare-cloud/pull/2644 to fix an unrelated issue.

The situation with `cpu_credits` is that that variable only means something on t3 instances, and on all other instances it's first set to standard and then pointlessly changed on the second run. If I continue to see this on the latest version of the aws provider plugin, I'll try to do something clever to set it based on whether the instances type starts with "t" or something like that.

**Pull request closed without merging**